### ISSUE TITLE
Incompatible error even though hardware is supported

### ISSUE THREAD

**Issue by rashidrafeek** [JuliaGPU/oneAPI.jl#168]
I am using a hardware which seems to be [supported](https://github.com/intel/compute-runtime#supported-platforms) (Processor with code name Coffee Lake):
```julia
julia> versioninfo()
Julia Version 1.7.2
Commit bf53498635 (2022-02-06 15:21 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 4
  JULIA_NUM_PRECOMPILE_TASKS = 8
```

But I am getting the following error while trying to use oneAPI:
```julia
julia> using oneAPI
┌ Error: No compatible oneAPI driver implementation found.
│ Your hardware probably is not supported by any oneAPI driver.
│ 
│ oneAPI.jl currently only supports the Intel Compute runtime,
│ consult their README for a list of compatible hardware:
│ https://github.com/intel/compute-runtime#supported-platforms
└ @ oneAPI.oneL0 ~/.julia/packages/oneAPI/BmdFb/lib/level-zero/oneL0.jl:95
```

**Comment by maleadt** [comment]
The problem is that the error thrown by oneAPI here can actually be caused by a variety of reasons. One can be that you don't have the necessary permissions to access the device nodes in `/dev/dri*`. Could you verify that? If that's the case, can you `strace` Julia doing `using oneAPI` and post that log?

**Comment by rashidrafeek** [comment]
Thanks for the info. I tried the same and it was a permission issue as you suggest. The issue was solved by adding myself to the `video` group following the instruction [here](https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-intel-oneapi-base-linux/top/before-you-begin.html#before-you-begin_GUID-9825D286-9B51-4B0F-8A7A-2541786B75C7). 

Would it be good to modify the error message to account for this case? If you feel its not worth it, feel free to close this issue.

**Comment by maleadt** [comment]
Yes, we should probably include that in the error message; I'll leave this issue open until we do.
